src/components/documentManager.ts

```diff
@@ -353,21 +353,19 @@ export abstract class DocumentManager {
      * @returns promise waiting for the backdrop to be set
      */
     private static async setRandomUserBackdrop(): Promise<void> {
-        const result = await JellyfinApi.authAjaxUser('Items', {
-            dataType: 'json',
-            query: {
-                ImageTypes: 'Backdrop',
-                IncludeItemTypes: 'Movie,Series',
-                Limit: 1,
-                MaxOfficialRating: 'PG-13',
-                Recursive: true,
-                SortBy: 'Random'
-                // Although we're limiting to what the user has access to,
-                // not everyone will want to see adult backdrops rotating on their TV.
-            },
-            type: 'GET'
+        const response = await getItemsApi(JellyfinApi.jellyfinApi).getItems({
+            imageTypes: ['Backdrop'],
+            includeItemTypes: ['Movie', 'Series'],
+            limit: 1,
+            maxOfficialRating: 'PG-13',
+            recursive: true,
+            sortBy: ['Random']
+            // Although we're limiting to what the user has access to,
+            // not everyone will want to see adult backdrops rotating on their TV.
```

Comment:
Feels like this comment should be above maxOfficialRating

Reply:
Agreed - it was at the bottom in the existing code so I just left it where it was but I'll shift it.